src/pipeline.js

```diff
@@ -215,8 +214,16 @@ class Pipeline {
    * @returns {Array.<Annotator>} annotators - those set for this pipeline
    */
   _getAnnotators() {
-    return this._getAnnotatorsKeys()
-      .map(annotatorKey => ANNOTATORS_BY_KEY[annotatorKey]);
+    return this._getAnnotatorsKeys().reduce((acc, annotatorKey) => {
+      if (
+        Object.prototype.hasOwnProperty.call(ANNOTATORS_BY_KEY, annotatorKey)
+      ) {
+        acc.push(ANNOTATORS_BY_KEY[annotatorKey]);
+      } else {
+        console.log(`Annotator "${annotatorKey}" doesn't exist or is not supported. Removing it from the pipeline.`);
```

Comment:
I don’t remember if I used console before, maybe throwing an error instead? It should not just warn you in that case, it should stop the execution as is a misconfiguration based on this wrapper limitations.

Reply:
Thanks for the quick feedback. Sounds absolutely reasonable! :) 
In that case, it is probably a good idea to dump the reduce alltogether and stick with the original map. 

```suggestion
 _getAnnotators() {
    return this._getAnnotatorsKeys()
      .map((annotatorKey) => {
        if (!Object.prototype.hasOwnProperty.call(ANNOTATORS_BY_KEY, annotatorKey)) {
          throw new Error(`Annotator "${annotatorKey}" doesn't exist or is not supported.`);
        }
        return ANNOTATORS_BY_KEY[annotatorKey];
      });
  }
```